### ISSUE TITLE
Feedback Capstone Project 3 - Purwadhika

### ISSUE THREAD

**Issue by bayuprasetyaa** [pratiwiastuti/Capstone-Project-3#1]
Video:
-	Video cukup oke
-	Bisa ditambahkan beberapa poin penjelasan (tidak hanya menampilkan gambar saja)
-	Video berupa ppt sesuai isi notebook. isinya mencakup bisnis understanding-conclusion, tapi analisisnya tidak sesuai.

Business Problem & Data Understanding:
-	context dan problem bisa lebih jelas. Maksudnya dapat meng-claim dan tidak dapat mengklaim itu gimana? kenapa harus ada penyaringan? selama ini prosesnya bagaimana? kenapa harus pakai machine learning?

Data Cleaning, Feature selection, & Feature Engineering:
-	Proses handling missing values pada kolom gender bisa lebih diperhatikan lagi. semua kolom yang terisi mempunyai value "Male". Jadi bisa saja baris yang kosong itu menunjukan nilai "Female" (tetapi perlu riset lagi)
-	Bisa ditambah penjelasan lebih detail mengenai anomali data pada kolom commision dan net sales. Seperti mengapa itu bisa terjadi, atau yang dianggap sebagai anomali mungkin sebenarnya memang ada penjelasannya (kasus spesial) dan memungkinkan untuk bisa terjadi
-	Ada beberapa proses yang tidak dijelaskan tentang alasan pemakaian metodenya. (seperti mengapa memakai smote untuk mengatasi imbalance?)
-	Penggunaan metode LabelEncoding untuk encoding kurang tepat (Lebih tepat digunakan untuk pelabelan variabel target, bukan features) https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelEncoder.html
-	Data nominal akan lebih baik menggunakan teknik One Hot encoding atau binary encoding. Menggunakan Label Encoding akan membuat hasil encoding seperti ordinal encoding.
-	Penggunaan sampling data seharusnya dilakukan pada data training saja. Bukan diterapkan pada keseluruhan dataset. Ini akan menyebabkan information leakage.
-	Duration -1 kemungkinan besar merupakan missing value. kenapa diubah jadi 1?

Analytics (Algorithm, Evaluation Metrics):
-	Hasil yang ditampilkan masih belum reliable karena handling imbalance yang kurang tepat dan belum menggunakan cross-validation untuk validasinya.
-	Tidak menggunakan pipeline
-	Tidak ada penjelasan limitasi model

Conclusion and Recommendation/Improvement:
-	Kesimpulannya sudah relevan dengan business problem, tapi masih terlalu sederhada (kurang detail)
-	Tidak ada penjelasan kapan model dapat dipercaya/tidak dapat dipercaya
-	Rekomndasi tidak actionable
